ocs_ci/ocs/ui/base_ui.py

```diff
@@ -816,6 +816,40 @@ def navigate_to_ocs_operator_page(self):
         logger.info("Enter the OCS operator page")
         self.do_click(self.generic_locators["ocs_operator"], enable_screenshot=False)
 
+    def navigate_to_odf_operator_page(self):
+        """
+        Function to navigate and select ODF operator under Installed Operators page
+
+        """
+        if (
+            self.ocp_version_semantic >= version.VERSION_4_9
```

Comment:
we don't need this if - else blocks on master branch. The 4.9 is deprecated and probably would not be ran again